### PR TITLE
Reduce RTC calls.

### PR DESCRIPTION
**Pull request by DaveDavenport**
Limit number off calls to rtc now, instead of as fast as possible.

### CODE REVIEW

File: software/wordclock/wordclock.ino [hunk 1]
```diff
@@ -10,7 +10,7 @@ RTC_DS3231 rtc;
 #ifdef NODO
 EnglishClockFace clockFace(ClockFace::LightSensorPosition::Top);
 // FrenchClockFace clockFace(ClockFace::LightSensorPosition::Top);
-// DutchClockFace clockFace(ClockFace::LightSensorPosition::Top);
+//DutchClockFace clockFace(ClockFace::LightSensorPosition::Top);
```
Comment (t0mg) [inline software/wordclock/wordclock.ino:13]: Unnecessary change :)

File: software/wordclock/wordclock.ino [hunk 2]
```diff
@@ -38,12 +38,19 @@ void setup()
   }
   display.setup();
   iot.setup();
+
 }
 
+long last_rtc_update =0;
```
Comment (t0mg) [inline software/wordclock/wordclock.ino:44]: Please add a whitespace after = sign ;)

**Review by t0mg — APPROVED** [review]

File: software/wordclock/wordclock.ino (hunk 1 above)
Comment (DaveDavenport) [reply software/wordclock/wordclock.ino:13]: Its consistent now with the "French" clock and the other side of the if/else block.
